container_checkpoint_test.go

```diff
@@ -20,18 +20,25 @@ package containerd
 
 import (
 	"bytes"
+	"context"
 	"fmt"
 	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
 	"strings"
 	"sync"
 	"syscall"
 	"testing"
 
 	"github.com/containerd/containerd/oci"
+	"github.com/containerd/containerd/runtime/linux/runctypes"
+	"github.com/containerd/containerd/runtime/v2/runc/options"
 )
 
 const (
 	v1runtime          = "io.containerd.runtime.v1.linux"
+	v2runtime          = "io.containerd.runc.v1"
```

Comment:
How about `io.containerd.runtime.v2.linux` ?

Reply:
@HusterWan this is the convention for the v2 runtime (https://github.com/containerd/containerd/pull/2434)